-1 
Tested at: 6b4786fa929f1eb155d35b89f8106f63990fb8bc
 I found an error when building: 
 
 <pre>>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-06-12-1100/src/RecoLocalCalo/HcalRecAlgos/src/HcalDeterministicFit.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-06-12-1100/src/RecoLocalCalo/HcalRecAlgos/src/HcalHFStatusBitFromDigis.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-06-12-1100/src/RecoLocalCalo/HcalRecAlgos/src/HcalHFStatusBitFromRecHits.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-06-12-1100/src/RecoLocalCalo/HcalRecAlgos/src/HcalHF_PETalgorithm.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-06-12-1100/src/RecoLocalCalo/HcalRecAlgos/src/HcalHF_S9S1algorithm.cc 
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-06-12-1100/src/RecoLocalCalo/HcalRecAlgos/src/HcalDeterministicFit.cc:6:44: error: 'constexpr' needed for in-class initialization of static data member 'const float HcalDeterministicFit::invGpar [3]' of non-integral type [-fpermissive]
 const float HcalDeterministicFit::invGpar[3];
                                            ^
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-06-12-1100/src/RecoLocalCalo/HcalRecAlgos/src/HcalDeterministicFit.cc:7:44: error: 'constexpr' needed for in-class initialization of static data member 'const float HcalDeterministicFit::negthre [2]' of non-integral type [-fpermissive]
 const float HcalDeterministicFit::negthre[2];
                                            ^
</pre> 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-9599/5469/summary.html

--- a/RecoLocalCalo/HcalRecAlgos/src/HcalDeterministicFit.cc
+++ b/RecoLocalCalo/HcalRecAlgos/src/HcalDeterministicFit.cc
@@ -3,6 +3,10 @@
 #include <climits>
 #include "RecoLocalCalo/HcalRecAlgos/interface/HcalDeterministicFit.h"
 
+constexpr float HcalDeterministicFit::invGpar[3];
+constexpr float HcalDeterministicFit::negthre[2];
+constexpr int HcalDeterministicFit::HcalRegion[2];
+
 using namespace std;
 
 HcalDeterministicFit::HcalDeterministicFit() {

